stdlib/public/stubs/Random.cpp

```diff
@@ -70,7 +70,8 @@ void swift_stdlib_random(void *buf, __swift_size_t nbytes) {
                                     static_cast<ULONG>(nbytes),
                                     BCRYPT_USE_SYSTEM_PREFERRED_RNG);
   if (!BCRYPT_SUCCESS(status)) {
-    fatalError(0, "Fatal error: 0x%.8X in '%s'\n", status, __func__);
+    fatalError(0, "Fatal error: 0x%lX in '%s'\n",
```

Comment:
Nit: does it make sense to use `lx` while here?

Reply:
I have no preference here regarding uppercase or lowercase. :)